libsolidity/interface/CompilerStack.cpp

```diff
@@ -103,6 +107,98 @@ CompilerStack::~CompilerStack()
 	TypeProvider::reset();
 }
 
+void CompilerStack::createAndAssignCallGraphs()
+{
+	for (Source const* source: m_sourceOrder)
+		if (source->ast)
+			for (ASTPointer<ASTNode> const& node: source->ast->nodes())
+				if (auto const* contract = dynamic_cast<ContractDefinition*>(node.get()))
+				{
+					Contract& contractState = m_contracts.at(contract->fullyQualifiedName());
+
+					contractState.contract->annotation().creationCallGraph = make_unique<CallGraph>(
+						FunctionCallGraphBuilder::buildCreationGraph(
+							*contract
+						)
+					);
+					contractState.contract->annotation().deployedCallGraph = make_unique<CallGraph>(
+						FunctionCallGraphBuilder::buildDeployedGraph(
+							*contract,
+							**contractState.contract->annotation().creationCallGraph
+						)
+					);
+
+					auto contractDeps = ranges::views::concat(
```

Comment:
This should just read 
```
contractState.contract->annotation().contractDependencies = contract->annotation().creationCallGraph->get()->bytecodeDependency + contract->annotation().deployedCallGraph->get()->bytecodeDependency;
```
If not, please adjust the types.

Also can you *please* introduce a local variable for the annotation? You are using it five times...

Reply:
We don't have an `operator+()` for maps....